api/controller/block/index.ts

```diff
@@ -0,0 +1,23 @@
+import {Block, SortBlock} from 'shared/model/block';
+import BlockService from 'api/service/block';
+import Response from 'shared/model/response';
+import { reqGetBlocks } from 'api/controller/block/types';
+import { RPC } from 'api/utils/decorators';
+
+class BlockController {
+
+    @RPC('GET_BLOCKS_HISTORY')
+    public getBlocksHistory(data?: reqGetBlocks) {
+        const blocks: Response<Array<Block>> = BlockService.getMany(data);
+        return blocks.data;
```

Comment:
return `ResponseEntity` here. Not only pure data

Reply:
I agree